Sentinel/Classes/CrashDetection/CrashToolDetailsView.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  CrashToolDetailsView.swift
+//  Sentinel
+//
+//  Created by Zvonimir Medak on 10.03.2025..
+//
+
+import SwiftUI
+
+struct CrashToolDetailsView: View {
```

Comment:
I would rather change names of this to `CrashDetectionToolDetailsView`, the same goes for the `CrashToolView`. I'm trying to avoid nitpicking, but this is more about scoping the functionality and use the naming in that context.